lib/excon/test/plugin/server/puma.rb

```diff
@@ -4,7 +4,7 @@ module Plugin
       module Server
         module Puma
           def start(app_str = app, bind_uri = bind)
-            bind.host = bind_uri.host.gsub(/[\[\]]/, '')
+            #bind.host = bind_uri.host.gsub(/[\[\]]/, '')
```

Comment:
It was supposed to be deleted. I thought I was justified leaving  it when I hit 325,000 earned galactic credits in battle front, but I guess not.